app/views/layouts/_header.html.erb

```diff
@@ -154,7 +155,7 @@
       <% unless params[:action] == "register" || params[:action] == "signup" %>
         <ul class="nav navbar-nav navbar-right">
           <li class="dropdown">
-            <a onClick="$('#chat').toggle();$('#chat').append('<iframe width=\'400px\' height=\'500px\' style=\'border:0;\' src=\'https://chat.publiclab.org\'></iframe>');">
+            <a onClick="$('#chat').toggle();">
```

Comment:
Ok I will change it. But seems a lot of inline javascript here. Will try to move this to any suitable javascript  file.